app/views/jobseekers/job_applications/_job_application_review_sections.html.slim

```diff
@@ -1,17 +1,20 @@
-= render "jobseekers/job_applications/review/personal_details", r: r, job_application: job_application
-= render "jobseekers/job_applications/review/professional_status", r: r, job_application: job_application
-= render "jobseekers/job_applications/review/qualifications", r: r, job_application: job_application
-= render "jobseekers/job_applications/review/training_and_cpds", r: r, job_application: job_application
-= render "jobseekers/job_applications/review/professional_body_memberships", r: r, job_application: job_application
-= render "jobseekers/job_applications/review/employment_history", r: r, job_application: job_application
-= render "jobseekers/job_applications/review/personal_statement", r: r, job_application: job_application
-- case job_application.vacancy.religion_type
-  - when "catholic"
-    = render "jobseekers/job_applications/review/catholic_religious_information", r: r, job_application: job_application
-  - when "other_religion"
-    = render "jobseekers/job_applications/review/non_catholic_religious_information", r: r, job_application: job_application
-= render "jobseekers/job_applications/review/references", r: r, job_application: job_application
-- if jobseeker_signed_in?
-  = render "jobseekers/job_applications/review/equal_opportunities", r: r
-= render "jobseekers/job_applications/review/ask_for_support", r: r, job_application: job_application
-= render "jobseekers/job_applications/review/declarations", r: r, job_application: job_application
+- if job_application.is_a?(UploadedJobApplication)
+  = render "jobseekers/job_applications/uploaded_job_applications/review/personal_details", r: r, job_application: job_application
+- else
```

Comment:
it would be nice to put a dynamic dispatch method on the 2 subclasses rather than check the type? WDYT?